File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int number = 0;
    int y = 1;
    int i = 0;

    // Get input from user

    while (number == 0) {
        printf("Height:  ");
        scanf("%d", &number);
        if (number <= 0 || number > 8) {
            number = 0;
        }
    }

    // Print output
    while (y <= number) {
        // 1. Print Spaces
        for (i = 0; i < number - y; i++) {
            printf(" ");
        }
        // 2. Print Blocks
        for (i = 0; i < y; i++) {
            printf("#");
        }
        // 3. Print Central Spaces
        printf("  ");
        // 4. Print Blocks
        for (i = 0; i < y; i++) {
            printf("#");
        }
        // 5. Print Spaces
        // printf(" ");
        // Down a level
        printf("\n");
        y++;
    }
}

File: sentimental-mario-more/mario.py
# Get User Input

while True:
    try:
        height = int(input("Height: "))
        if (height > 0 and height < 9):
            break
    except:
        print("enter an integer maaaaatteeee!")

for y in range(1, height+1):
    # Print Spaces
    i = 0
    while (i < height - y):
        print(" ", end="")
        i += 1

    # Print Blocks
    i = 0
    while (i < y):
        print("#", end="")
        i += 1

    # Print Central Spaces
    print("  ", end="")

    # Print Blocks again
    i = 0
    while (i < y):
        print("#", end="")
        i += 1

    # Now actially print a new line
    print("")